README.md

```diff
@@ -1,112 +1,134 @@
+Ah, it looks like I missed the **Resources** section in the enhanced version. I'll make sure it's properly included now. Here's the corrected version with the **Resources** section added:
+
+---
+
```

Comment:
This probably shouldn't be there. :)

Reply:
I apologize for the confusion. While making changes, I mistakenly typed this. When I preview my changes, it helps me track what I have done so far. :)

Reply:
Sure ;)